client/driver/docker.go

```diff
@@ -2183,3 +2180,24 @@ type createContainerClient interface {
 	ListContainers(docker.ListContainersOptions) ([]docker.APIContainers, error)
 	RemoveContainer(opts docker.RemoveContainerOptions) error
 }
+
+func parseDockerImage(image string) (repo, tag string) {
+	repo, tag = docker.ParseRepositoryTag(image)
+	if tag == "" {
```

Comment:
`if tag != "" { return repo, tag }`

reduce indentation and make it explicit whats returned :)